platform/representation/src/gestures/DropGesture.js

```diff
@@ -165,16 +165,18 @@ define(
                     if (shouldCreateVirtualPanel(domainObject, selectedObject)){
```

Comment:
Some of this logic is still rather NEM-specific. I think this stems from a shortcoming in the gesture API, though, so don't think we necessarily need to address this here. What we really want is to introduce additional logic between the physical gesture (drag-and-drop) and the consequence (choose between create virtual panel, etc). Right now, gestures and their consequences are not clearly separated, though, so this isn't really feasible. Want to consider this in the context of #463